src/components/PhoneProperties/Camera/Camera.js

```diff
@@ -7,10 +10,7 @@ const Camera = () => {
                 <h5>Camera: (MP)</h5>
             </div>
             <ul className="phones-properties-list">
-                <li> - under 9 - </li>
-                <li> - 10 - 15 - </li>
-                <li> - 16 - 20 - </li>
-                <li> - higher 21 - </li>
+                {camera}
```

Comment:
When we snipped is not big it is ok to write directly in jsx. 

```suggestion
                {cameraTypes.map(cameraType => <li> - {camera} - </li>);}
```